assets/src/domain/eventEditor/services/apollo/queries/prices/useFetchPrices.ts

```diff
@@ -37,6 +41,25 @@ const useFetchPrices = (skipFetch: boolean = null): FetchQueryResult<PricesList>
 		},
 	});
 
+	useEffect(() => {
+		// if poll interval has changed
+		if (pollInterval !== prevPollInterval) {
+			// if poll interval has been set/changed
+			if (pollInterval > 0) {
```

Comment:
can we combine these 2 in a variable?
```
const pollIntervalHasChanged = pollInterval !== prevPollInterval && pollInterval > 0;
```

Reply:
no because the second conditional has an else clause that is also a subclause of the first conditional

Reply:
There is also an else block which will then be executed everytime.